domain/domain.go

```diff
@@ -52,6 +52,7 @@ type Domain struct {
 	statsHandle     unsafe.Pointer
 	statsLease      time.Duration
 	ddl             ddl.DDL
+	info            *infoSyncer
```

Comment:
The first letter of infoSyncer is lower-case, and it is used in the package server, it is really weird.